TicketManagement/src/main/java/com/example/TicketManagement/controller/EventController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.TicketManagement.controller;
+
+import com.example.TicketManagement.model.Event;
+import com.example.TicketManagement.service.implementation.EventService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class EventController {
+    private final EventService eventService;
+
+    public EventController(EventService eventService) {
+        this.eventService = eventService;
+    }
+
```

Comment:
For handling exceptional cases you can look into @ExceptionHandler or @ControllerAdvice (a bit more complex) annotations.